course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -382,7 +382,7 @@
                 $http.get(url)
                     .success(function(data, status, headers, config, statusText) {
                         //check if the right data was obtained before storing it
-                        if (data.course_instance_id === id){
+                        if (data.course_instance_id == id){
```

Comment:
Sorry,  I am to blame for this last minute change that I tested via Jasmine but not via the tool. Good catch